services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -70,6 +72,10 @@
     @JsonProperty(RegistryManagementConstants.FIELD_MAPPER)
     private String mapper;
 
+    @JsonProperty(RegistryManagementConstants.FIELD_AUTHORITIES)
```

Comment:
It is not returned if it is not set, but with the change this property may be set via API (and subsequently it will be returned then). 

Hence I included in the specification.